packages/cli/__tests__/zosfiles/__integration__/create/vsam/__snapshots__/cli.files.create.dataset.vsam.integration.test.ts.snap

```diff
@@ -65,10 +65,6 @@ exports[`Create VSAM Data Set should display the help 1`] = `
       you do not specify a secondary allocation, a value of ~10% of the primary
       allocation is used.
 
-   --show-attributes  | --pa (boolean)
```

Comment:
Should the new option `--attributes` be added in this snapshot?

Reply:
i skipped vsam because i wasn't using it 😄 - it's a simple change though.  i'll add it to this pr